assets/sass/components/_applications.scss

```diff
@@ -242,14 +240,21 @@ $statuses: (
 
     @include mq('medium') {
         display: flex;
-        justify-content: space-between;
+
+        .application-card__summary {
+            flex: 1 1 auto;
+        }
+
+        .application-card__progress {
+            flex: 0 0 40%;
```

Comment:
This is the replacement for `flex-basis: 60%;` above, setting the opposite column instead.